content/6_Advanced/String_Search.problems.json

```diff
@@ -262,6 +262,18 @@
         "year": 2008
       }
     },
+    {
+      "uniqueId": "cf-888G",
```

Comment:
string search???
trie isnt just for strings yknow

Reply:
it is, right? integers are binary strings

Reply:
benq said to put it here

Reply:
mmmmmm ok